cypress/component/Modal.spec.tsx

```diff
@@ -485,10 +486,8 @@ context(`given the [Components/Popups/Modal, Custom focus] story is rendered`, (
       });
 
       it('should have an aria-labelledby attribute', () => {
-        cy.findByRole('dialog', {name: 'Acknowledge License'}).should(
-          'have.attr',
-          'aria-labelledby'
-        );
+        cy.get('[role="dialog"]').should('exist');
```

Comment:
I don't think `get` makes a difference here, can we change these back to `findByRole`?